packages/shared/src/modules/ai-assistant/libs/enums/ai-assistant-api-path.enum.ts

```diff
@@ -0,0 +1,11 @@
+const AiAssistantApiPath = {
+	ACCEPT_TASK: "/chat/accept-task",
+	ADD_MESSAGE: "/chat/add-message",
+	CHANGE_TASK: "/chat/change-task",
```

Comment:
```suggestion
	CHAT_ACCEPT_TASK: "/chat/accept-task",
	CHAT_ADD_MESSAGE: "/chat/add-message",
	CHAT_CHANGE_TASK: "/chat/change-task",
```
Sane for others. Key should be similar to the value